content/d3/concepts/scaling/terms/domain/domain.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: '.domain()'
+Description: 'denotes minimum and maximum values of the input data'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'D3'
+  - 'functions'
+CatalogContent:
+  - 'learn-d3'
+  - 'paths/computer-science'
+---
+
+The **`.domain()`** method is associated with scale functions in `D3.js`. This method is used to specify the input domain of a scale, which defines the range of values that the scale should map from. In other words, it sets the minimum and maximum values that the scale will accept as input.
+
+## Syntax
+
+```pseudo
+d3.scaleLinear().domain([minValue, maxValue]).range([minOutput, maxOutput]);
+
+```
+
+- `scaleLinear`: Construct a continuous linear scale where input data (domain) maps to specified output range.
+- `range`: Output range that are mapped from input values.
+
+## Example
+
+In the example given below the `scale` can map values within the specified domain to corresponding values within the range. The `scale` is set to have a domain from `0` to `100` and range from `0` to `500`. When a value is passed within a given domain to the scale, it will map to corresponding values within specified range.
```

Comment:
```suggestion
In the example given below the `scale` will map values within the specified domain to the corresponding range given. The `scale` is set to have a domain from `0` to `100` and range from `0` to `500`. When a value is passed within a given domain to the scale, it will map them accordingly.
```